### PR TITLE
Access control adjustments

### PR DESCRIPTION
**Pull request by Twixes**
## Changes

Adjustments for https://github.com/PostHog/posthog/pull/3486.

## Checklist

-   [x] Jest tests

### CODE REVIEW

File: src/sql.ts [hunk 1]
```diff
@@ -1,35 +1,36 @@
 import { Plugin, PluginAttachmentDB, PluginConfig, PluginConfigId, PluginError, PluginsServer } from './types'
 
+function pluginConfigsInForceQuery(specificField?: string): string {
```
Comment (macobo) [inline src/sql.ts:3]: Suggestion: replace `string` with `'id' | 'plugin_id'`. This makes the following string substitution a lot less scary from an injection standpoint.

File: src/utils.ts [hunk 2]
```diff
@@ -388,6 +388,14 @@ export async function createRedis(serverConfig: PluginsServerConfig): Promise<Re
     return redis
 }
 
+export function pluginDigest(plugin: Plugin): string {
```
Comment (macobo) [inline src/utils.ts:391]: This seems dead

File: src/sql.ts (hunk 1 above)
Comment (Twixes) [reply src/sql.ts:3]: Good idea, I used `keyof PluginConfig` for a bit of flexibility

File: src/utils.ts (hunk 2 above)
Comment (Twixes) [reply src/utils.ts:391]: Merge conflict resolution gone wrong, fixed

**Pull request merged**